Berksfile

```diff
@@ -0,0 +1,7 @@
+source 'https://supermarket.getchef.com'
```

Comment:
I fixed the matcher for the definition and there were no chefspec tests in the cookbook which will use that matcher for validation and in order for me to write those chefspec tests I had to use fake cookbook where that definition was used in the recipe in order to let chefspec see that cookbook I had to define that in berksfile.